golang/internal/grpc/grpc.go

```diff
@@ -589,44 +593,44 @@ func executeContainerCommand(ctx context.Context, command *common.ContainerComma
 
 func streamContainerLog(reader ContainerLogReader,
 	client agent.Agent_ContainerLogClient,
-	containerID string,
+	prefix, name string,
```

Comment:
Isn't this the same function that is used for inspecting containers on a node?
Because if that is the case we can't make it sure that a container has a prefix. Either we should not display containers that are not managed by dyrectorio or show logs for other "external" containers too.

Reply:
In that case the prefix is a "" string